.github/labeler.yml

```
@@ -6,15 +6,15 @@
+"Area: JSON":
 - changed-files:
   - all-globs-to-any-file: ['**/*.json']
 
-"Audio":
+"Area: Audio":
 - changed-files:
   - any-glob-to-any-file: ['**/snd_*', '**/*.ogg', '**/*.wav']
 
-"Sprites":
+"Area: Sprites":
 - changed-files:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)